sources/packages/backend/apps/api/src/services/application-offering-change-request/application-offering-change-request.service.ts

```diff
@@ -91,6 +91,7 @@ export class ApplicationOfferingChangeRequestService {
         "programYear.id",
         "currentAssessment.id",
         "educationProgram.id",
+        "educationProgram.isActive",
```

Comment:
Is this relevant to get eligible applications for application offering change. ??

My understanding is, If a completed application has deactivated program, it can still be requested for application offering change.